_posts/2023-11-23-stork-latest-news.adoc

```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+title: 'SmallRye Stork Unwrapped: Exploring New Features and Enhancements'
+date: 2023-11-23
+tags: extension smallrye-stork
+synopsis: Explore in detail the latest features added to Smallrye Stork
+author: aureamunoz
+---
+:imagesdir: /assets/images/posts/smallrye-stork
+Since its initial release in January 2022, Stork has undergone significant development, introducing new features that extended its capabilities and improved developer experience.
+
+This blog post takes a deep dive into the evolution of SmallRye Stork beyond its initial release, providing a detailed exploration of its fresh additions.
```

Comment:
```suggestion
This blog post takes a deep dive into the evolution of SmallRye Stork beyond it's initial release, providing a detailed exploration of its fresh additions.
```

Reply:
Right, after thinking about it once more yes that's the correct form. Non-native speaker here.

Reply:
No problem. Thank you again